src/core/InputDropdown/style.ts

```diff
@@ -161,22 +161,19 @@ const square = (props: InputDropdownProps): SerializedStyles => {
 
   return css`
     border-radius: ${corners?.m}px;
-    height: 34px;
```

Comment:
was this doing anything before?

Reply:
The height of this component should be implicitly calculated from the line height and padding rather than hard-coded.